.github/workflows/build.yml

```diff
@@ -115,7 +113,7 @@ jobs:
           push: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') }}
       -
         name: GitHub Release
-        uses: softprops/action-gh-release@v1
+        uses: crazy-max/ghaction-github-release@v1
```

Comment:
Is this a fork of the other one? Wondering why it had to be switched

Reply:
It looks unmaintained and still use Node 12, see https://github.com/docker/buildx/pull/1418

Reply:
ugh

Reply:
Oh! Looks like they just updated https://github.com/softprops/action-gh-release/pull/275

Reply:
Ohhh damn few hours after my fork :sweat_smile: